pwiz_tools/Skyline/Model/DocSettings/SrmSettings.cs

```diff
@@ -1064,14 +1070,15 @@ public static IonMobilityFilter GetSonarMzIonMobilityFilter(double mz, double wi
             LibraryIonMobilityInfo libraryIonMobilityInfo,
             double ionMobilityMax)
         {
+            IonMobilityFilter result = null;

```

Comment:
Would it work to just use IonMobilityFilter.Empty here? Then you could just skip all the result != null logic and return result. Wouldn't IonMobilityFilter.Empty fail result.HasIonMobilityValue?

Reply:
Yes, but this works just as well, and is more robust in case anything in between does return a null.